core/templates/domain/subtopic_viewer/ReadOnlySubtopicPageObjectFactory.ts

```diff
@@ -25,19 +25,36 @@ import {
   SubtopicPageContents,
   SubtopicPageContentsObjectFactory
 } from 'domain/topic/SubtopicPageContentsObjectFactory';
+import { ISubtopicBackendDict, Subtopic, SubtopicObjectFactory } from
+  'domain/topic/SubtopicObjectFactory';
 
 export interface ISubtopicDataBackendDict {
   'subtopic_title': string;
   'page_contents': ISubtopicPageContentsBackendDict;
+  'next_subtopic': ISubtopicBackendDict | null,
+  'topic_id': string
 }
 
 export class ReadOnlySubtopicPageData {
+  parentTopicId: string;
   subtopicTitle: string;
   pageContents: SubtopicPageContents;
+  nextSubtopic: Subtopic | null;
 
-  constructor(subtopicTitle: string, pageContents: SubtopicPageContents) {
+  constructor(
+      parentTopicId: string,
+      subtopicTitle: string,
+      pageContents: SubtopicPageContents,
+      nextSubtopic: Subtopic| null
```

Comment:
For consistency, add space before | too.

Reply:
Done.